WBooks/LibraryViewController.swift

```diff
@@ -60,35 +64,31 @@ extension LibraryViewController: UITableViewDelegate, UITableViewDataSource {
     }
     // Cell generator
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: LibraryCell.xibFileCellName, for: indexPath) as? LibraryCell else {
-            print("Error on dequeueReusableCell")
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: LibraryCell.xibFileCellName) as? LibraryCell else {
             return UITableViewCell()
         }
-
-        // Fill in the cell with info
         
-        let dict = bookArray[indexPath.section]
-
-        cell.imageBook.image = UIImage(named: dict["img"]!)
-        cell.topLabel.text = dict["title"]
-        cell.botLabel.text = dict["author"]
+        let book: Book = libraryViewModel.books.value[indexPath.section]
         
-        return cell
-    }
-    
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+        cell.imageBook?.image = UIImage()   // Add grey frame to make loading prettier
         
-        let navigationBar = self.navigationController?.navigationBar
+        if let url = book.imageUrl {
+            cell.imageBook?.load(url: url)
+        }
         
-        navigationBar?.setBackgroundImage(UIImage(), for: .default)
-        navigationBar?.shadowImage = UIImage()
-        navigationBar?.backgroundColor = .clear
-        navigationBar?.isTranslucent = true
-        navigationBar?.tintColor = UIColor.white
+        cell.topLabel?.text = book.title
+        cell.botLabel?.text = book.author
```

Comment:
If this is the bottom label, we should rename it to `bottomLabel` (it's a good practice to use full names 😉)